### DISCUSSION TITLE
New Enterprise accounts have metered billing for Git LFS

### DISCUSSION THREAD

**Discussion by goldieowner17** [community/community#126753] [Enterprise]
Enterprise accounts on GitHub.com, created after June 2, 2024, along with organizations owned by these accounts, have access to the enhanced billing platform. This includes enhanced billing for Git Large File Storage (LFS). Enterprises who participated in the beta program also have access to this platform. Other Enterprise accounts on GitHub.com, and Free, Pro, and Team accounts, will gain access to the enhanced billing platform in the coming months.

The enhanced billing platform transitions Git LFS from a pre-paid, quota-based model (data packs) to a post-paid, usage-based model (metered billing). This new platform offers better spending control and detailed visibility, allowing for a clearer understanding of your usage with more granular controls.

**Comment by Murazaki** [comment]
Great news for indie game development and machine learning !
What about the 2GiB push limit ? Is that going to be changed ?

**Reply by davidstaheli** [reply]
Hi @Murazaki. The 2 GiB push limit isn't being changed as part of this LFS metered billing update. But we agree that it would greatly help with game development and storing asset files, etc. via LFS. We're looking into ways to increase the 2 GiB limit. Unfortunately, we don't have a timeframe for it yet.

**Reply by Murazaki** [reply]
Hi David, thanks for the info. We would really like a way to fix or circumvent that, as it actually makes us split commits when it's not making much sense regarding commit history.

**Reply by davidstaheli** [reply]
@Murazaki, I can imagine what a hassle that must be. Thanks for mentioning this.
